python/arcticdb/arctic.py

```diff
@@ -38,40 +38,88 @@ def __init__(self, uri: str, encoding_version: EncodingVersion = EncodingVersion
+                |                           |                                                                                                                                                               |
+                |                           | Default certificate path in various Linux distributions:                                                                                                      |
+                |                           | "/etc/ssl/certs/ca-certificates.crt"                  Debian/Ubuntu/Gentoo etc.                                                                               |
+                |                           | "/etc/pki/tls/certs/ca-bundle.crt"                    Fedora/RHEL 6                                                                                           |
+                |                           | "/etc/ssl/ca-bundle.pem"                              OpenSUSE                                                                                                |
+                |                           | "/etc/pki/tls/cacert.pem"                             OpenELEC                                                                                                |
+                |                           | "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"   CentOS/RHEL 7                                                                                           |
+                |                           | "/etc/ssl/cert.pem"                                   Alpine Linux                                                                                            |
+                +---------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+                Note: Support for Azure Blob Storage is currently only available in *non-Conda* binaries distribution.
+
+                Exception: Azure exceptions message always ends with `{AZURE_SDK_HTTP_STATUS_CODE}:{AZURE_SDK_REASON_PHRASE}`.
+
+                Please refer to https://github.com/Azure/azure-sdk-for-cpp/blob/24ed290815d8f9dbcd758a60fdc5b6b9205f74e0/sdk/core/azure-core/inc/azure/core/http/http_status_code.hpp for
+                more details of provided status codes. Note that due to a bug in Azure C++ SDK (https://github.com/Azure/azure-sdk-for-cpp/issues/4738), Azure may not give meaningful status codes and
+                reason phrases in the exception. To debug these instances, please set the environment variable ``export AZURE_LOG_LEVEL`` to ``1`` to turn on the SDK debug logging.
+
+
+            LMDB
+            ----
+
+                The LMDB URI connection scheme has the form ``lmdb:///<path to store LMDB files>``. There are no options
+                available for the LMDB URI connection scheme.
```

Comment:
The length of this now makes it impractical to view in an IDE and an annoyance if the stack trace included this function in Pytest (which may dump the full source).

It's more sensible to move this in a docs page now.

Reply:
I'll change the line in this PR - would rather move in another PR as this PR is blocking the next release.

Reply:
https://github.com/man-group/ArcticDB/issues/565